src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -376,6 +379,29 @@ internal PSKeyword GetLogKeywords()
             return defaultValue;
         }
 
+        FileStream WaitForFile(string fullPath, FileMode mode, FileAccess access, FileShare share)
+        {
+            const int MaxTries = 5;
```

Comment:
nit: local variable shouldn't have capital first character.

Reply:
It is a constant.